### ISSUE TITLE
Production errors with Svelte

### ISSUE THREAD

**Issue by stephenlrandall** [mauriciopoppe/function-plot#204]
I'm attempting to use this library with [Svelte](https://svelte.dev)([Kit](https://kit.svelte.dev)). In development, I immediately run into a pair of solvable issues:

- `global is not defined` -- Either set `global = window` in a `<svelte:head>` tag, or set `define: { global: "window" }` in the Vite config file
- `Module "events" has been externalized for ...` -- Separately install the "events" package.

Everything then works in development mode (although it would be nice if these were fixed at the function-plot level).

In production mode, I'm running into an unavoidable `Attempting to define property on object that is not extensible.` error. Digging into the Svelte output, the specific line it's complaining about is:

```
var out2 = Object.assign(relational, arithmetic, algebra, trigonometric);
```

since `relational` is defined (not sure in which dependency...) as:

```
const relational = /* @__PURE__ */ Object.freeze(/* @__PURE__ */ Object.defineProperty({
...
}, Symbol.toStringTag, { value: "Module" }));
```

So the issue is the `Object.freeze` call on something that's later a target in `Object.assign`. (If I remove the Object.freeze call on `relational` in the built output, everything works fine.) Where is this coming from, and is there a way that this issue can be addressed?

**Comment by stephenlrandall** [comment]
Could you just spread the objects instead?

```
const out = { ..._Interval, ...constants, ...round, ... };
```

**Comment by mauriciopoppe** [comment]
Thanks for the suggestion, I'll try that and see if the doc generator works with that

**Comment by mauriciopoppe** [comment]
I've fixed the issue downstream in https://github.com/mauriciopoppe/interval-arithmetic/releases/tag/v1.0.7 and published `function-plot@1.22.9-0`, please try with that prerelease version and let me know how it goes

**Comment by mauriciopoppe** [comment]
If this is still an issue feel free to reopen this bug
